packages/core/src/context.js

```diff
@@ -32,6 +45,9 @@ let withEmotionCache = function withEmotionCache<Props>(
       </EmotionCacheContext.Consumer>
     )
   }
+
+  render.displayName = getDisplayName<Props>(func)
```

Comment:
does `func` have a name under any circumstances? I think we always pass anonymous functions to it

Reply:
Not before, so I made `injectDisplayName` in `styled-base` to inject displayName before `withEmotionCache`, 
it still has no `name` property, but maybe post in the future